ZupIT/ritchie-cli
# Default tree refactoring to use file manager of package stream

**- What I did**

In this PR we exchange the use of `io/ioutil` and `/fileutil` to `FileReadExister`  in all functions of `default_tree.go` file.

**- How to verify it**

Check the file `default tree.go` to see the difference and in the tests of the related file check the use of interface.

**- Description for the changelog**

Default tree refactoring to use file manager of package stream
